README.md

```diff
@@ -129,3 +129,11 @@ SimpleCov.start
 ```
 
 You can see your test coverage when you run your tests. If you want this in a graphical form, uncomment the `HTMLFormatter` line and see what happens!
+
```

Comment:
You could update your README with your progress so far, and what steps you'd take next.